fermipy/plotting.py

```diff
@@ -1,15 +1,7 @@
-from __future__ import absolute_import, division, print_function, \
-    unicode_literals
-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function
 import copy
 import os
-import matplotlib
-
-try:
-    os.environ['DISPLAY']
-except KeyError:
-    matplotlib.use('Agg')
-
 import matplotlib.pyplot as plt
```

Comment:
Usually configuring the matplotlib backend is something that should be left up to the end user, not done automatically by libraries.

For non-interactive plotting, what you do here is the right thing to do:
```python
import matplotlib
matplotlib.use('agg')
```

I have that (and other plotting configuration) at the top of every non-interactive script that makes plots.
This can be explained to users in the docs.

Doing it automatically and sometimes on `import fermipy.plotting` can be confusing or annoying.
So I'm +1 to remove this. But I don't feel strongly, if you like it that way let me know and I'll put it back.